entrypoint.sh

```diff
@@ -40,7 +42,11 @@ end
 coms = github.issue_comments(repo, pr_number)
 
 if check_duplicate_msg == "true"
-  duplicate = coms.find { |c| c["body"] == message }
+  if duplicate_msg_pattern != nil
+    duplicate = coms.find { |c| (c["body"] =~ (Regexp.new duplicate_msg_pattern)) != nil }
```

Comment:
I'll merge this as-is because it's not a big deal, but since you mentioned in the description that you're new to Ruby, fwiw you don't need the `!= nil`s here, since anything non-nil (well, except `false`) will be truthy.